practice-two/src/styles/base/typography.css

```diff
@@ -1,12 +1,12 @@
 @font-face {
-  font-family: 'roboto';
+  font-family: "roboto";
   src: url('../../assets/fonts/roboto/Roboto-Regular.ttf') format('truetype');
   font-weight: 400;
   font-display: swap;
 }
 
 @font-face {
-  font-family: 'lucida grande';
+  font-family: "lucida grande";
   src: url('../../assets/fonts/lucida-grande/LucidaGrandeBold.ttf') format('truetype');
```

Comment:
single quote

Reply:
I don't get it. Did you want me to use single quote or vice versa?

Reply:
`I will choose double quotes for all of css files`. and your coding `  src: url('../../assets/fonts/lucida-grande/LucidaGrandeBold.ttf') format('truetype');` Is it clear?

Reply:
My fault for not having checked the code clearly. I've just updated it